src/Compilers/CSharp/Portable/CSharpResources.resx

```diff
@@ -4666,4 +4666,7 @@ To remove the warning, you can use /reference instead (set the Embed Interop Typ
   <data name="ERR_PublicSignButNoKey" xml:space="preserve">
     <value>Public signing was specified and requires a public key, but no public key was specified.</value>
   </data>
-</root>
+  <data name="ERR_InvalidReal" xml:space="preserve">
+    <value>Invalid real literal.</value>
```

Comment:
We are definitely not consistent today; I believe there is an issue to improve consistency.

Having said that, I've been using the convention of adding a period when the error message is a sentence. In this case it is just a noun phrase. I could go either way.